py/tracks/latest.toml

```diff
@@ -48,5 +48,6 @@ full = [
     "../specs/in_subquery.toml",
     "../specs/in_subquery.toml",
     "../specs/union.toml",
-    "../specs/select/disabled_doc_values.toml"
+    "../specs/select/disabled_doc_values.toml",
+    "../specs/select/hyperloglog.toml"
```

Comment:
minor: add a trailing comma, [toml supports that](https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.4.0.md#array). it has the nice side effect that when you add a line, the previous line does not need to be changed :)